ytbdownloader.py

```diff
@@ -34,25 +34,25 @@ def load_config(filename):
 
 def download(config):
     """Start download process by processing configuration""" 
-    for i in range(len(config)):
+    for i in range(len(config['tasks'])):
         # Check if 'enable' tag is true
```

Comment:
Could you please use appropriate local variables to work with json tags? It would make code much cleaner here.
For example:
```python
tasks = config('tasks')
task = tasks[i]
isEnabled =  task['enable']
...
```

Reply:
added